### ISSUE TITLE
Past time can not be grasped

### ISSUE THREAD

**Issue by Mike162** [kimai/kimai#1103]
Good Morning all,
I installed Kimai yesterday and I am thrilled.
It is a great relief especially in teamwork.

There are still project times from 2017 to be added.
This was started yesterday, unfortunately, these are not displayed.

On the other hand, in the MYSQL DB, the entries are available.

The file [kimai] /include/vars.php has been changed to from $ kga ['calender_start'] = ""; to $ kga ['calender_start'] = "05/01/2016"; set.

Unfortunately, the entries are not visible yet.

How can the condition be changed?

Sorry for the bad english I do not speak english.
Google translator had to help.

**Comment by Mike162** [comment]
Good Morning all,
I installed Kimai yesterday and I am thrilled.
It is a great relief especially in teamwork.

There are still project times from 2017 to be added.
This was started yesterday, unfortunately, these are not displayed.

On the other hand, in the MYSQL DB, the entries are available.

The file [kimai] /include/vars.php has been changed to from $ kga ['calender_start'] = ""; to $ kga ['calender_start'] = "05/01/2016"; set.

Unfortunately, the entries are not visible yet.

How can the condition be changed?

Sorry for the bad english I do not speak english.
Goole translator had to help.
